src/Core/RevEng.Core.80/PatchedSqlServerDatabaseModelFactory.cs

```diff
@@ -1492,6 +1491,7 @@ private void GetTriggers(DbConnection connection, IReadOnlyList<DatabaseTable> t
     [tr].[name] AS [trigger_name]
 FROM [sys].[triggers] AS [tr]
 JOIN [sys].[tables] AS [t] ON [tr].[parent_id] = [t].[object_id]
+LEFT JOIN [sys].[extended_properties] AS [ep] ON [ep].[major_id] = [t].[object_id] AND [ep].[minor_id] = 0 AND [ep].[class] = 1 AND [ep].[name] = 'microsoft_database_tools_support'
```

Comment:
Why was this added?

Reply:
because of the table filter that added anywhere